### PR TITLE
fix(tooltip): fix tooltip cannot hide when parent element has set sto…

### PR DESCRIPTION
**Pull request by chenc041**
…pPropagation

[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes `Tooltip` can not hide, when the parent element has been set stopPropagation

### Changelog
🇨🇳 Chinese
- Fix: 修复 Tooltip 在 React17 里如果父级有阻止点击事件冒泡弹出层收起会失效 [#593](https://github.com/DouyinFE/semi-design/issues/593) [@chenc041](https://github.com/chenc041) 

---

🇺🇸 English
- Fix: Fixed Tooltip in React17 if the parent prevents the click event from bubbling and the pop-up layer is collapsed will fail  [#593](https://github.com/DouyinFE/semi-design/issues/593) [@chenc041](https://github.com/chenc041) 

### Checklist
- [x] Test or no need
- [ ] Document or no need
- [x] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

**Comment by chenc041** [comment]
解决方案和 `Calendar`,  `Cascader` 解决方案保持一直

File: packages/semi-ui/tooltip/index.tsx [hunk 1]
```diff
@@ -326,11 +326,11 @@ export default class Tooltip extends BaseComponent<TooltipProps, TooltipState> {
                         cb();
                     }
                 };
-                document.addEventListener('click', this.clickOutsideHandler, false);
+                document.addEventListener('click', this.clickOutsideHandler, {capture: true});
             },
             unregisterClickOutsideHandler: () => {
                 if (this.clickOutsideHandler) {
-                    document.removeEventListener('click', this.clickOutsideHandler, false);
+                    document.removeEventListener('click', this.clickOutsideHandler, {capture: true});
```
Comment (shijiatongxue) [inline packages/semi-ui/tooltip/index.tsx:333]: 麻烦新增一个测试用例测试这个表更 💖

Comment (shijiatongxue) [reply packages/semi-ui/tooltip/index.tsx:333]: Jest 如果不合适（Enzyme 可能对事件的处理有缺陷）的话可以用 Cypress 测试下

Cypress 可以参考下这里，有问题随时交流

https://github.com/DouyinFE/semi-design/blob/main/cypress/README.md

Comment (chenc041) [reply packages/semi-ui/tooltip/index.tsx:333]: 👌🏻

Comment (chenc041) [reply packages/semi-ui/tooltip/index.tsx:333]: 补上啦

**Pull request merged**